.travis.yml

```diff
@@ -1,2 +1,7 @@
 language: node_js
 script: npm run test
+node_js:
```

Comment:
why do we have this here?

Reply:
@9jaswag By default, Travis was installing a very old version of node, so I had to specify it to use version 10.